docs/deployment/distribution_component.md

```diff
@@ -0,0 +1,45 @@
+---
+id: distribution_component
+title: Cumulus Distribution Terraform Module
+hide_title: true
+---
+
+# Cumulus Distribution Terraform Module
```

Comment:
Didn't we say we're going to put an example in here?

Reply:
Or link to an example?

Reply:
https://github.com/nasa/cumulus/pull/1086/commits/cb567628f9274df0e62d34c855a0b94a85139b8b